src/coreclr/pal/src/configure.cmake

```diff
@@ -21,7 +21,11 @@ elseif(NOT CLR_CMAKE_TARGET_FREEBSD AND NOT CLR_CMAKE_TARGET_NETBSD)
 endif()
 
 if(CLR_CMAKE_TARGET_LINUX AND NOT CLR_CMAKE_TARGET_ANDROID)
-  set(CMAKE_RT_LIBS rt)
+  if(NOT CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
Probably would be better to reverse condition and use `if(CLR_CMAKE_HOST_ARCH_LOONGARCH64)`

Reply:
Thanks for your review!